packages/api-clients/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "pagopa-interop-api-clients",
+  "version": "1.0.0",
+  "private": true,
+  "description": "PagoPA Interop Api Clients",
+  "main": "dist",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "scripts": {
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:autofix": "eslint . --ext .ts,.tsx --fix",
+    "format:check": "prettier --check src",
+    "format:write": "prettier --write src",
+    "start": "node --loader ts-node/esm -r 'dotenv-flow/config' --watch ./src/index.ts",
+    "build": "tsc",
+    "check": "tsc --project tsconfig.check.json",
+    "generate-model": "node --loader ts-node/esm ./generate.ts",
+    "clean-generated": "pnpm exec rm ./src/generated/*.ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@pagopa/eslint-config": "3.0.0",
+    "@types/node": "20.14.6",
+    "handlebars": "^4.7.7",
```

Comment:
```suggestion
    "handlebars": "4.7.7",
```

Reply:
This is recurring in many PRs since when we use Renovate that pins dependencies - could it make sense to configure `pnpm` to avoid adding the `^` by default when saving a new dependency?

Reply:
if possible that would be helpful